notebooks/_06-Architecture.md

```diff
@@ -152,38 +169,45 @@ N/A
 - **Flyte?**
 - **Make?**
 - **Other?**  
-We don't have any plans for this. please make any suggestions as you see needed/fit.
+Developing a project workflow file that shows how all specified requirements of each workflow
+step are met. 
 
 ## 8. Description of experimentation protocols (if relevant):
 **If experiments need to be run (e.g. hyper-parameter tuning, model-selection, A/B tests, etc.):**
 ### 8.1 How will experiments be launched?
-N/A
+We can store launch the experiment from a jupyter notebook, and **hopefully** from the command line.
 
 ### 8.2 How will experiment reproducibility be ensured?
-N/A
+Simulated data, as well as any experiment output will be store either in csv/json/pickle file
+based on the nature of the object (dataset, model, etc..) and will be loaded into a notebook.
+A third party library called `sacred` can be used https://sacred.readthedocs.io/en/stable/
 
 ### 8.3 How will experiment meta-data (e.g. launch configurations) be stored?
-N/A
+The meta-data for the experiment could be stored in a requirements.txt, the source code,
+and parameters stored in a .json/pickle/csv file. the sacred library has a `run` object
+that stores all experiment info and configuration.
 
 ### 8.4 How will experiment-created data (e.g. results) be stored?
-N/A
+The experiment-created data will be stored in json/pickle/csv files.
 
 ### 8.5 How will experiment-created data be analyzed?
-N/A
+Experiment metrics will be analyzed to find statistics about means and variances of parameters
+to make sure they fall within the expected ranges.
 
 ### 8.6 How will experiment analyses be prepared for public reporting? 
-N/A
+Experiment analyses will be stored in .json/csv files and used to create plots summarizing
+results of the experiments.
 
 ## 9. Description of user-interface:
 ### 9.1 How will users (including myself) interact with the system that is built?
 - **Will there be a command line interface?**
 - **Will users be editing configuration files, and if so, where will they be stored?**
 - **Will there be ability to replay the DAG representing one’s data analysis?**
 
-Users will use a jupyter notebook to interact with the system that is build
-
-No configuration files will need to be edited. If any functions need to be edited, one could easily make the adjustment
-to the source code.
+Users will use a jupyter notebook to interact with the system. The `papermill` library
+will be used to parametrize a notebook. Parameters of the notebook could also be saved
+in a .json file that is loaded into the notebook. For experiments, the `sacred` library
```

Comment:
Smooth. I looked into sacred before but for whatever reason never fell in love with it. I just checked it out again and it looks like it'll be good!